samples/Chat/src/app/App.tsx

```diff
@@ -52,6 +54,8 @@ export default (): JSX.Element => {
             setDisplayName={setDisplayName}
             setThreadId={setThreadId}
             setEndpointUrl={setEndpointUrl}
+            /* @conditional-compile-remove(rich-text-editor-composite-support) */
+            setIsRichTextEditorEnabled={setIsRichTextEditorEnabled}
```

Comment:
Seems like this api only controls the Sendbox not the Editbox? If so, should we rename it to richTextSendBox?

Reply:
Hmmm it should control both. We set richtexteditor in the message thread api in the chatcomposite.

Could be a bug. Let me investigate

Reply:
Seems to be working for the edit box for me. Do you have repro steps?